### ISSUE TITLE
Users not created

### ISSUE THREAD

**Issue by sg0894074** [danilop/LambdAuth#55]
After installing, no users were able to sign up.
Page displays 'user Not created', when looking at the CloudWatch logs for the LambdAuthCreateUser function, I found this error:

2016-10-05T18:15:38.131Z    b72f1452-8b27-11e6-a0ef-3f70cb5b1481
{ "errorMessage": "Error in sendVerificationEmail: MessageRejected: Email address is not verified. The following identities failed the check in region US-WEST-2: xxxxxxxxx@xxxxx.com"}

Recipients of the verification email, that is the end users, should not have to be registered on SES for it to send them email. 

When trying with a registered account, it worked the first time, then the second one I got this other error:

2016-10-05T18:19:17.803Z    3a41c422-8b28-11e6-969a-41fcce75977a
{"errorMessage": "Error in sendVerificationEmail: InvalidParameterValue: Domain ends with dot"}

This user had the same domain as I did and had entered its address correctly

**Comment by sg0894074** [comment]
Please disregard the second error, it was on our end :)
The first one is still valid though. I am forced to manually verify the users on DynamoDB before they can login

**Comment by aggied** [comment]
I'm having the same error. CloudWatch reports: 

```
{
    "errorMessage": "Error in sendVerificationEmail: MessageRejected: Email address is not verified. The following identities failed the check in region US-WEST-2: xxxxx@xxxxx.com"
}
```

any thoughts?

**Comment by aggied** [comment]
For what it's worth --

I just tested signing up an email from an SES _verified_ domain, and it looks like the registration worked. So it seems the problem is that somewhere along the way only emails from SES verified domains are acceptable... 

Has this script always worked like this? I'm a first time user

**Comment by aggied** [comment]
So... as it turns out the particular AWS SES account I'm using is still in the SES "sandbox," which I forgot about. @sg0894074 perhaps this is our problem? http://docs.aws.amazon.com/ses/latest/DeveloperGuide/request-production-access.html

**Comment by sg0894074** [comment]
Yes, that seems to be my problem too. I was not aware of this limitation by default. Thanks!
